task1/TaskMethods.cs

```diff
@@ -0,0 +1,361 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task1
+{
+    public class TaskMethods
+    {
+        //
+        //Calculator
+        //
+        public static void GetCalculation()
+        {
+            double firstNum = GetNumber();
+            double secondNum = GetNumber();
+            string sign = GetSign();
+            double result = Calculator(firstNum, secondNum, sign);
+            WriteResult(firstNum, secondNum, sign, result);
+        }
+
+        private static string GetSign()
+        {
+            Console.WriteLine("Input sign of action +, -, *, /");
+            string sign = Console.ReadLine();
+            if (sign != "+" && sign != "-" && sign != "*" && sign != "/")
+            {
+                Console.WriteLine("Input data is not valid");
+                throw new ArgumentException(String.Format($"{sign} is not +, -, *, /"));
+            }
+            else
+                return sign;
+        }
+
+        private static double GetNumber()
+        {
+            Console.WriteLine("Input number");
+            bool success = Double.TryParse(Console.ReadLine(), out double number);
+            if (success == false)
```

Comment:
No need to compare bool type with false or true, we can use negotiation:
if(!success)
  ConsoleWriteLine()